README.md

```diff
@@ -87,9 +90,10 @@ make build-and-push-must-gather
 To collect data for custom repositories for Open Data Hub set the following variables:
 
 ```
-export OPERATOR_NS= <name-for-operator-namespace>
-export NOTEBOOKS_NS= <name-for-notebooks-namespace>
-export MONITORING_NS= <name-for-monitoring-namespace>
-export APPLICATIONS_NS= <name-for-applications-namespace>
+export OPERATOR_NS=<name-for-operator-namespace>
+export NOTEBOOKS_NS=>name-for-notebooks-namespace>
+export MONITORING_NS=>name-for-monitoring-namespace>
+export APPLICATIONS_NS=<name-for-applications-namespace>
+export MODELS_REGISTRY_NAMESPACE=<name-for-model-registry-namespace>
```

Comment:
```suggestion
export MODEL_REGISTRIES_NAMESPACE=<name-for-model-registries-namespace>
```
Minor spelling nit

Reply:
let me get it into https://github.com/red-hat-data-services/must-gather/pull/68